### DISCUSSION TITLE
New release 2.6.0

### DISCUSSION THREAD

**Discussion by newerton** [jmcdo29/nest-commander#543] [General]
I updated to version 9 of NestJs, and it is in conflict with version 2.5.0, I saw that in the master branch it was updated, do you have any date for a new release?

```bash
npm ERR! While resolving: nest-commander@2.5.0
npm ERR! Found: @nestjs/common@9.0.3
...
npm ERR! node_modules/@nestjs/common
npm ERR!   peer @nestjs/common@"^8.0.0" from nest-commander@2.5.0
npm ERR!   node_modules/nest-commander
npm ERR!     nest-commander@"^2.5.0" from the root project
```

**Comment by jmcdo29** [comment]
I'm gonna make this a part of 3.0.0 actually with a couple of other issues I'm working on fixing. Hoping to have it ready by the end of the week

**Reply by jmcdo29** [reply]
Technically the project should work on v9, you just have to tell the package manager to not worry about peer deps

**Reply by newerton** [reply]
@jmcdo29 Thanks for fast feedback. It is working perfectly in version 9. Using with --force, or --legacy-peer-deps.

**Comment by jomarocas** [comment]
any update on the new version? or how can I collaborate on something?

**Reply by jmcdo29** [reply]
I've opened up this [~~draft~~ PR](https://github.com/jmcdo29/nest-commander/pull/548), there's still work to be done for the full 3.0.0 release, but if you want to watch it progress you're more than welcome to. I'll try to get it out sometimes this week, granted I've got the time to make all the feature updates
